### DISCUSSION TITLE
Do delayed start and early end work in CUI promt?

### DISCUSSION THREAD

**Discussion by miasik** [Comfy-Org/ComfyUI#493] [Q&A]
My prompt templates in a1111 have constructions like these:

[key:4] — my image stays untouched by the key until step 4 finished.
[key::4] — the key stops working after step 4.

And a more complex construction:
`[[[(key:1.4):(key:1.3):1]:(key:1.2):3]:(key:1.1):5]`

It means that the key's weight gets changed with steps. It decreases in this example.
- 1.4 only for the first steps
- 1.3 from step 2 and 3
- 1.2 from step 4 and 5
- 1.1 from 6 and to the end

[Details](https://www.reddit.com/r/StableDiffusionInfo/comments/ylp6ep/some_detailed_notes_on_automatic1111_prompts_as/)

Do such constructions work in CUI?
I know that there's a node Advanced sampler, but it looks like an over hit for my task, as for me.

**Comment by comfyanonymous** [comment]
The functionality to sample only certain steps with certain settings is exposed with KSamplerAdvanced. It's much more powerful because it actually lets you do things like switch models for different sampling steps.

**Reply by miasik** [reply]
Yes, I've already appreciated the trick with two models and I was impressed so much!
Started trying to imagine how such a construction with KASes will look like.

Constant part of my negative + changing part x4 + merging x4 + KAS x4
It becomes much, much harder if I have more than one construction like that. And yes, I do. There are at least two complex constructions in my general negative.

I'm totally in panic.
